Markdown/developer_product_guide/Embedded_Software.md

```diff
@@ -1,25 +1,24 @@
 # Embedded Software
 
-##  Scade One
+## Scade One
 
-Scade One is the new generation of SCADE (see below). In the Scade One environment you use the swan language. Scade One also has a PyAnsys OSS Python library called PyScadeOne and a developer portal landing page as well.
+Scade One is the new generation of SCADE. In the Scade One environment, you use the Swan language. Scade One also includes PyScadeOne, an open-source Python library.
 
-- [Scade One for Developers](https://developer.ansys.com/docs/scade-one)
-- [PyScadeOne](https://github.com/ansys/pyscadeone)
-- [PyScadeOne docs](https://scadeone.docs.pyansys.com/)
+- [Scade One for developers (Developer portal documentation)](https://developer.ansys.com/docs/scade-one)
+- [PyScadeOne repository (PyAnsys)](https://github.com/ansys/pyscadeone)
```

Comment:
@jgd10 Remove this line. I don't think another product classification page provides a link to the GitHub repo.